metaflow/plugins/argo/argo_workflows.py

```diff
@@ -143,6 +143,37 @@ def _sanitize(name):
         # allowed by Metaflow - guaranteeing uniqueness.
         return name.replace("_", "-")
 
+    @classmethod
+    def remove_schedule(cls, name):
```

Comment:
Nit: I guess it's an existing pattern - could be `@staticmethod` instead - we don't use `cls` here (but we don't use `cls` for other existing methods either.

Reply:
good call. I went with the existing pattern but see no reason not to break the habit here, so changed to staticmethods instead :)